bot/wuestbot.py

```diff
@@ -1,6 +1,374 @@
-#!/usr/bin/env python3
+import os
+import logging
+from traceback import print_exc
 
-from app import run
+import flask
+import requests
+from telebot import TeleBot, types
+from dotenv import load_dotenv
 
-if __name__== "__main__":
-	run()
\ No newline at end of file
+from app.database import db_objects
+
+# set logger
+logging.basicConfig(filename='log.txt', encoding='utf-8', level=logging.DEBUG)
+
+# load env variables from .env file
+load_dotenv()
+API_TOKEN = os.getenv('API_TOKEN', '')
+HOST = os.getenv('HOST', '')
+PORT = os.getenv('PORT', '8443')
+ADMINS = os.getenv('ADMINS', '')
+
+# URL to let telegram know where we are 
+WEBHOOK_URL = f"https://{HOST}:{PORT}/bot"
+
+bot = TeleBot(API_TOKEN)
+app = flask.Flask(__name__)
+
+img_dir = os.path.abspath('../bot/app/img')
+
+# Process webhook calls
+@app.route(f'/bot', methods=['POST'])
+def webhook():
+    if flask.request.headers.get('content-type') == 'application/json':
+        json_string = flask.request.get_data().decode('utf-8')
+        update = telebot.types.Update.de_json(json_string)
+        bot.process_new_updates([update])
+        return ''
+    else:
+        flask.abort(403)
+
+# Handle '/start' and '/help'
+@bot.message_handler(commands=['help', 'start'])
+def send_welcome(message):
+    bot.reply_to(message, message.text)
+
+# Handle all other messages
+@bot.message_handler(func=lambda message: True, content_types=['text'])
+def echo_message(message):
+    bot.reply_to(message, message.text)
+#
```

Comment:
Let's delete this whole comment, we can always fetch the code from previous commits